test/Test/Pos/Block/Logic/Mode.hs

```diff
@@ -102,7 +102,9 @@ sudoLiftIO m = BaseMonad (liftIO m)
 
 instance MonadIO BaseMonad where
     liftIO m = BaseMonad . liftIO $ do
-        putStr ("Don't do IO in tests! " :: String)
+        -- if you see a lot of stars in the test log, a hunting season for
+        -- not-mocked operations is open.
+        putStr ("*" :: String)
```

Comment:
lol, can you explain why this solution is better than `putStr ("Don't do IO in tests! " :: String)`? just meme?